fibonacci.php

```diff
@@ -0,0 +1,22 @@
+<?php   
+function Fibonacci($number){ 
+    if ($number < 0)
+        return "Wrong number!";
+	
+    if ($number == 0) 
+        return 0;     
+    else if ($number == 1) 
+        return 1;     
+      
+    // recursive call to get the upcoming numbers
+    else
+        return (Fibonacci($number-1) +  
+                Fibonacci($number-2)); 
+} 
+  
+// Example usage
+$number = 15;
+for ($counter = 1; $counter < $number; $counter++){   
+    echo Fibonacci($counter),' '; 
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'Fibonacci'.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6353686)